### DISCUSSION TITLE
Is there a way to disable dark mode toggling in homepage?

### DISCUSSION THREAD

**Discussion by ymc9** [facebook/docusaurus#9083] [Q&A] [answered]
Making the homepage work with both light and dark themes can be challenging. Is there a way to disable it there but keep the capability for the docs and blog parts? Using Docusaurus V2.

**Comment by slorber** [comment, marked as answer]
We don't have that kind of feature as far as I know.

Not sure it's a good idea to implement considering presenting a theme toggle on the homepage and having it to nothing would also be quite confusing for users.

**Reply by jrschumacher** [reply]
Can we get the post by @omonk marked as the answer?

**Reply by slorber** [reply]
for some weird reason, GitHub will not allow me to do that, and only @ymc9 can

Note that the feature was asking how to disable the switch ONLY for the homepage, not the whole site. I think your use case is different, and my answer remains accurate.

**Comment by omonk** [comment]
For anyone else here before checking the docs like me, you can configure this in the preset options

```
  colorMode: {
      defaultMode: 'light',
      disableSwitch: false,
      respectPrefersColorScheme: false,
    },
```

**Reply by jrschumacher** [reply]
https://docusaurus.io/docs/api/themes/configuration#color-mode---dark-mode
